OWNERS

```diff
@@ -1,14 +1,12 @@
 reviewers:
   - bparees
-  - mfojtik
-  - soltysh
   - csrwng
   - sayan-biswas
   - divyansh42
+  - flavianmissi
 approvers:
   - adambkaplan
   - bparees
-  - mfojtik
-  - soltysh
   - sayan-biswas
+  - flavianmissi
```

Comment:
I suggest bringing this up in a sync call to see who is actually knowledgeable versus who has been assigned bug fixes and there is no longer someone knowledgeable to merge them.

Reply:
Yup, will do. Removed my own name for now.